core/templates/packages.el.template

```diff
@@ -1,31 +1,63 @@
-;;; packages.el --- %LAYERNAME% Layer packages File for Spacemacs
+;;; packages.el --- %LAYER_NAME% layer packages file for Spacemacs.  -*- lexical-binding: t; -*-
```

Comment:
I'm not for introducing lexical binding in the file variables simply because the current layers don't  have it, so let's keep it this way.